cores/esp8266/FS.cpp

```diff
@@ -175,6 +233,22 @@ bool Dir::next() {
     return _impl->next();
 }
 
+bool Dir::rewind() {
```

Comment:
It may be *technically* correct, but *logically* I'd have to disagree.  The externally visible state is changed by this (i.e. a pointer inside an ope directory), even if this class's own pointer to that encapsulated class doesn't change.

The pointer doesn't change in all other methods as well, and they're none of them `const`.